README.md

```diff
@@ -0,0 +1,705 @@
+
+#### Step: Commit and push *Containerfiles/version_major/Containerfile.** changes
+
+> The step is skipped if '*Push to production registries*' is not checked (`inputs.production` set to `false`.)
+
+Uses [EndBug/add-and-commit@v9](https://github.com/marketplace/actions/add-commit) to commit and push Containerfiles, which was downloaded on the previous step, and changed by the previous job.
+
+The commit message is:
+```yaml
+AlmaLinux ${{ inputs.version_major }} image build as of ${{ needs.build.outputs.date_stamp }} (with ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}).
+```
+It includes AlmaLinux version major, image build date, and reference to this Github Action.
+
+## Publish images to the Docker Library
+
+Tree illustration of the workflow Jobs and Steps for AlmaLinux 9 minimal image:
+```
+Publish images to the Docker Library
+│
+├── 8 default definition preparing
+.
+.
+.
+├── 8 minimal definition preparing
+.
+.
+.
+├── 9 default definition preparing
+.
+.
+.
+├── 9 minimal definition preparing
+│   ├── Set up job
+│   ├── Checkout container-images, branch 'docker-library'
+│   ├── Checkout official-images, branch 'master'
+│   ├── Get need data for the definition
+│   ├── Render the definition
+│   ├── Upload the definition for 9 minimal
+│   ├── Post Checkout official-images, branch 'master'
+│   ├── Post Checkout container-images, branch 'docker-library'
+│   └── Complete job
+│
+└── Create Pull Request with the new definition file
+    ├── Set up job
+    ├── Checkout official-images, branch 'master'
+    ├── Sync official-images with docker-library/official-images
+    ├── Download all definitions
+    ├── Create head of official-images/library/almalinux
+    ├── Merge definitions into official-images/library/almalinux
+    ├── Prepare date stamp
+    ├── Prepare time stamp
+    ├── Commit and push official-images/library/almalinux
+    ├── Create Pull Request for official-images/library/almalinux
+    ├── Post Checkout official-images, branch 'master'
+    └── Complete job
+```
+
+### Inputs
+
+The workflows inputs are:
+- `pr` - boolean *'Publish to the Docker Official Library'* with the default value `true` (checked). The input indicates whether to create Pull Request to Docker with AlmaLinux *Definition File*.
+
+- `draft` - boolean '*Draft Pull Request*' with the default value `false` (not checked). The input indicates whether the [Pull Request is draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) or it is ready to review.
+
+### Job: *version_major* *image_types* definition preparing
+
+Job iterates (using matrix) with AlmaLinux all `version_major`, and `image_types` (`default` and `minimal`). Multiple jobs run simultaneously for each of versions and for each of image types.
```

Comment:
Job iterates (using matrix) with AlmaLinux all `version_major`, and `image_types` (`default` and `minimal`). Multiple jobs run simultaneously for each of the versions and each of the image types.

Reply:
Done.